views/scss/qti/_choice.scss

```diff
@@ -227,8 +250,8 @@
     }
     .real-label{
         input{
-            position: absolute;
-            left: -10000px;
+            opacity: 0;
+            left: auto;
```

Comment:
It's gonna left space(input size) since container wraps an element. I would suggest to leave hidden styles as it was before, or could you explain why it changed?

Reply:
It is to avoid extreme long scrollbar for RTL, and actually I simplify this rule this should be enough to hide original checkbox
<img width="1904" alt="Screenshot 2021-07-26 at 14 28 07" src="https://user-images.githubusercontent.com/2067027/126994332-e60e81e8-4775-4e84-becb-31d8c93be6eb.png">